Fix aliases documentation and incorporate more feedback from 'Brunch and learn' session on the topic (#396)
grafana/writers-toolkit: docs/sources/write/front-matter/index.md

For an alias in the page `/docs/grafana/latest/alerting/manage-notifications/`:

> - The current directory element (`.`) refers to the directory containing the current page, _not_ the directory of the current page.

Reviewer: ```suggestion
- The current directory element (`.`) refers to the parent directory of the current page, _not_ the directory that contains the current page.  An easy way to remember is that one dot means you have moved one step back in the file path to the page.
```

Author: I think we should avoid "easy" if possible (https://developers.google.com/style/word-list#easy).

My only concern with this tip is that it is only true at the start of the alias, and that's really because the starting directory for any alias is the directory containing the current page.

I'm worried people will internalize `.` goes back one part and `..` goes back two when in actually `.` goes nowhere and `..` goes back one part.

Reviewer: Ah, good point about "easy".  

Describing how aliases work is hard!

  In this example, this is the page `/docs/grafana/latest/alerting/`.

> - The parent directory element (`..`) refers to the parent directory of the current directory element.

Reviewer: ```suggestion
- The parent directory element (`..`) refers to the parent directory of the current directory element. An easy way to remember is that two dots means you have moved two steps back in the file path to the page.
```

  In this example, this is the page `/docs/grafana/latest/`.

> The following table demonstrates how to redirect **FROM PAGE**, to **TO PAGE**.

Reviewer: ```suggestion
In the following table, the **FROM PAGE** column lists the page that requires a redirect, for example because the page has been moved or no longer exists. The **TO PAGE** is where we want to redirect them, for example the renamed page or where the content has been moved.
The third column shows the alias that should be added to the front matter of the file in the **TO PAGE** column to create the proper redirect.
```

Reviewer: I don't think so.

Author: I've attempted to rewrite this bit using a list of the columns and your suggestions for explaining each column in https://github.com/grafana/writers-toolkit/pull/396/commits/462fe219617fcc67146f0b876b4f5cc947dbfc9a
